setup.py

```diff
@@ -77,8 +76,11 @@
         'Topic :: Text Processing :: General',
     ],
     packages=find_packages(),
-    install_requires=[],
-    package_data={'': ['*.dat', '*.crt']},
+    python_requires='>=3.7',
+    install_requires=[
+        'importlib_resources >= 1.3  ; python_version < "3.9"',
```

Comment:
Type checkers don't have great support for the `try`/`except` style of providing backwards compatibility. So instead of relying on `pkg_resources`, which may be missing even for Python versions 3.7 and 3.8 if a tool like `uv` was used to setup the environment, it seemed better to just rely on the `importlib_resources` backport with the minimum version that supports the `files` function. That way we don't have a soft-dependency on `setuptools`.